google/cloud/bigtable/instance_resource.h

```diff
@@ -0,0 +1,86 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_BIGTABLE_INSTANCE_RESOURCE_H
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_BIGTABLE_INSTANCE_RESOURCE_H
+
+#include "google/cloud/bigtable/version.h"
+#include "google/cloud/project.h"
+#include "google/cloud/status_or.h"
+#include <iosfwd>
+#include <string>
+
+namespace google {
+namespace cloud {
+namespace bigtable {
+GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
+
+/**
+ * This class identifies a Cloud Bigtable Instance.
+ *
+ * A Cloud Bigtable instance is identified by its `project_id` and
```

Comment:
Maybe just repeat what it says here, and link this document:

https://cloud.google.com/bigtable/docs/instances-clusters-nodes

Reply:
Done.